packages/playground/src/types.ts

```diff
@@ -537,3 +543,68 @@ export interface FunctionSchedulesFilter {
   createdTime?: Date;
   cronExpression?: string;
 }
+
+/* ************************
+   Annotations API types
+************************ */
+
+export type AnnotatedResourceType = 'file';
+export type AnnotationType =
```

Comment:
Perhaps it is sbetter to not have this as a type for now and just let the user input a String, as we have currently no good way to fill in the possible values automatically.
Then, when we have a proper integration with the `annotation-types` repo, we can create this type/enum automatically and it has all the available values.

Reply:
Agreed; changed.